application/client/src/app/ui/tabs/module.ts

```diff
@@ -14,6 +15,7 @@ import { TimezoneSelectorModule } from '@elements/timezones/module';
         TabSourceDltStreamModule,
         TimezoneSelectorModule,
         TabSourceTextStreamModule,
+        TabSourceMultipleFilesModule,
```

Comment:
add `TabSourceMultipleFilesModule` also into `bootstrap` section bellow to prevent production bug